packages/frontend/src/Layout.tsx

```diff
@@ -1,10 +1,28 @@
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
 import Container from '@mui/material/Container';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
 import { Outlet } from 'react-router-dom';
 
 export default function Layout() {
     return (
         <>
             <Container maxWidth="lg" component="main" role="main">
+                <Box sx={{ flexGrow: 1 }}>
+                    <AppBar position="static">
```

Comment:
AppBar should not in Container but outside.
An appbar can be in head section, Container is main section

Reply:
An appComponent was created in an other package. 
It's now imported in Layout.tsx and outside the container.